ExternalModule.php

```diff
@@ -35,6 +35,12 @@ function hook_every_page_top($project_id) {
             }
         }
 
+        // if features are present then hover field note.
+        if (!empty($features)) {
+            $js_vars['hover_field_note'] = getFieldsWithActionTags();
```

Comment:
The idea is to create another action tag (something like: `@FIELD-NOTE-DISPLAY="hover"`).